components/Counter/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styled from "styled-components";
+
+//Counter component
+export default function Counter(props) {
+  return <StyledCounter>+</StyledCounter>;
```

Comment:
Is it really a counter? Looks like just a div with a `+` sign.
And you accepting the `props` object, but you are not using it. You can omit then the parameter.